src/modules/tools/zprobe/CartGridStrategy.cpp

```diff
@@ -349,14 +424,14 @@ bool CartGridStrategy::handleGcode(Gcode *gcode)
 
 // These are convenience defines to keep the code as close to the original as possible it also saves memory and flash
 // set the rectangle in which to probe
-#define LEFT_PROBE_BED_POSITION (0)
+#define LEFT_PROBE_BED_POSITION (x_start)
 #define RIGHT_PROBE_BED_POSITION (x_size)
 #define BACK_PROBE_BED_POSITION (y_size)
```

Comment:
Ensure that the x_size and y_size used here are correct. They are absolute points.